bin/check-bundle-size.js

```diff
@@ -55,10 +55,11 @@ log('File sizes look OK.');
 
 function isTooBig({ path, size }) {
   // Special cases:
-  const simpleName = basename(path).replace(/-[\w-]{8}\./, '.');
+  const simpleName = basename(path).replace(/(-[\w-]{8})+\./, '.');
   switch (simpleName) { // eslint-disable-line default-case
     case 'index.js':             return size >   450_000;
-    case 'web-form-renderer.js': return size > 3_000_000;
+    case 'web-form-renderer.js': return size > 3_300_000;
```

Comment:
Now `renderer` is being stripped off `web-form-renderer`, since it's exactly 8 characters, so matches the regex above. 😭 Maybe we could just change web-form-renderer.js here:

```suggestion
    case 'web-form.js': return size > 3_300_000;
```